app/index.js

```diff
@@ -2,13 +2,15 @@ import { setup } from './insights.js'
 import 'log-timestamp'
 import { startSfdMessageReceiver, stopSfdMessageReceiver } from './messaging/index.js'
 import server from './server.js'
+import logger from './logger.js'
 
 const init = async () => {
-  await startSfdMessageReceiver(server.logger)
+  await server.register([logger])
```

Comment:
We tend to initialise server plugins in the server file along with the rest of it's bootstrapping. That way you can initialise a server for testing and it'll be sure it's the same as what we use when running the app. Means you wont have to keep stubbing the logger methods.

It's configured to log silently during tests 🙊 https://github.com/DEFRA/ffc-ahwr-sfd-messaging-proxy/blob/main/app/logger.js#L4

Reply:
Might be worth a quick call this one, was hoping you'd highlight this.